modules/reactor/test/tests/meshgenerators/core_mesh_generator/gold/core_hex_2d_datadriven_out.json

```diff
@@ -0,0 +1,244 @@
+{
+    "reporters": {
+        "metadata": {
+            "type": "MeshMetaDataReporter",
+            "values": {
+                "values": {
+                    "type": "std::map<std::string, RestartableDataReporter::Value, std::less<std::string>, std::allocator<std::pair<std::stringconst, RestartableDataReporter::Value>>>"
+                }
+            }
+        }
+    },
+    "time_steps": [
+        {
+            "metadata": {
+                "values": {
+                    "MeshMetaData/amg1/assembly_type": {
+                        "type": "unsigned short",
+                        "value": 1
+                    },
+                    "MeshMetaData/amg1/background_block_name": {
+                        "type": "std::vector<std::string>",
+                        "value": []
+                    },
+                    "MeshMetaData/amg1/background_region_id": {
+                        "type": "std::vector<unsigned short>",
+                        "value": [
+                            3
+                        ]
+                    },
+                    "MeshMetaData/amg1/duct_block_names": {
+                        "type": "std::vector<std::vector<std::string>>",
+                        "value": []
+                    },
+                    "MeshMetaData/amg1/duct_halfpitches": {
+                        "type": "std::vector<double>",
+                        "value": [
+                            1.7703
+                        ]
+                    },
+                    "MeshMetaData/amg1/duct_region_ids": {
+                        "type": "std::vector<std::vector<unsigned short>>",
+                        "value": [
+                            [
+                                4
+                            ]
+                        ]
+                    },
+                    "MeshMetaData/amg1/extruded": {
+                        "type": "bool",
+                        "value": false
+                    },
+                    "MeshMetaData/amg1/is_homogenized": {
+                        "type": "bool",
+                        "value": false
+                    },
+                    "MeshMetaData/amg1/is_single_pin": {
+                        "type": "bool",
+                        "value": false
+                    },
+                    "MeshMetaData/amg1/pin_block_name_map": {
+                        "type": "std::map<unsigned short, std::vector<std::vector<std::string>>, std::less<unsigned short>, std::allocator<std::pair<unsigned short const, std::vector<std::vector<std::string>>>>>",
+                        "value": []
```

Comment:
shouldnt there be a key of 1 here? empty value is correct I think

Reply:
Good catch, this metadata gets ignored in data driven mode so it is not initialized with any meaningful values unless RGMB is generating an output mesh. I've gone ahead and removed this metadata entry from being defined in data driven mode. Same for "pin_region_id_map" metadata entry